### PR TITLE
add PINT Stake Pool

### PR DESCRIPTION
**Pull request by curtindoavida**
### Commitment

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist

I have:
- [x] Added a telegram handle of an **individual** and not that of a group or channel.
- [x] Added a social media account with activity.
- [x] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [x] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [x] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -5315,5 +5315,15 @@
...
+ },
+ {
+   "ticker": "PINT",
+   "poolId": "11152917b1e01cb25ac9d993b207bfed11c097b34b77a5ce6f97cfa9",
+   "social": {
+       "twitter": "PINT_token",
+       "website": "https://beerhouse.io/",
+       "discord": "pintbeerhouse#6023"
+   },
+   "telegram": "pintbeerhouse"
```
Comment (Bez625) [inline registry.json:5327]: You need to include the `operatorName` and `github` fields even if blank

File: registry.json [hunk 2]
```diff
@@ -5315,5 +5315,15 @@
    "telegram": "CassandradeVries",
    "operatorName": "",
    "github": "NEVdataDyne"
+ },
+ {
+   "ticker": "PINT",
+   "poolId": "11152917b1e01cb25ac9d993b207bfed11c097b34b77a5ce6f97cfa9",
+   "social": {
+       "twitter": "PINT_token",
+       "website": "https://beerhouse.io/",
```
Comment (Bez625) [inline registry.json:5324]: You need to include the `facebook` , `instagram` and `youtube` fields even if blank

Comment (curtindoavida) [reply registry.json:5313]: done

**Review by Bez625 — DISMISSED** [review]

File: registry.json [hunk 3]
```diff
@@ -5305,6 +5305,21 @@
...
+       "discord": "pintbeerhouse#6023"
+       "facebook": "",  
+       "instagram": "",
+       "youtube": ""
+   },
+   "telegram": "pintbeerhouse",
+   "operatorName": "",
+   "github": ""
+  }
```
Comment (Bez625) [inline registry.json:5322]:
```suggestion
  },
```

Comment (Bez625) [reply registry.json:5322]: You need to make this change as it wont let me. The json is invalid otherwise

**Review by Bez625 — APPROVED** [review]

File: registry.json [hunk 4]
```diff
@@ -5305,6 +5305,21 @@
    "operatorName": "",
    "github": "NEVdataDyne"
  },
+ {
+   "ticker": "PINT",
+   "poolId": "11152917b1e01cb25ac9d993b207bfed11c097b34b77a5ce6f97cfa9",
+   "social": {
+       "twitter": "PINT_token",
+       "website": "https://beerhouse.io/",
+       "discord": "pintbeerhouse#6023"
```
Comment (Bez625) [inline registry.json:5314]:
```suggestion
       "discord": "pintbeerhouse#6023",
```

Comment (Bez625) [reply registry.json:5314]: Another missing comma sorry

**Review by Bez625 — CHANGES_REQUESTED** [review]
Please fix json

**Comment by curtindoavida** [comment]
Do I need to do anything else?

**Review by Bez625 — APPROVED** [review]

**Pull request merged**